socorro/unittest/conftest.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from markus.testing import MetricsMock
+import pytest
+
+
+@pytest.fixture
+def metricsmock():
+    """Returns MetricsMock that a context to record metrics records
+
+    Usage::
+
+        def test_something(metricsmock):
+            with metricsmock as mm:
+                # do stuff
+                assert mm.has_record(
+                    stat='some.stat',
+                    kwargs_contains={
+                        'something': 1
+                    }
+                )
+
+    """
+    return MetricsMock()
```

Comment:
I remember you saying you didn't like this kind of fixture. In order to switch it to be a context manager, we need to fix Markus. I wrote up this for that:

https://github.com/willkg/markus/issues/29

Once I/you/someone implements that, we can push out a new Markus and then change this and the one or two uses of it.